src/nanoemoji/color_glyph.py

```diff
@@ -327,13 +327,15 @@ def create(ufo, filename, glyph_id, codepoints, picosvg):
             base_glyph.unicode = next(iter(codepoints))
 
         # Grab the transform + (color, glyph) layers for COLR
-        painted_layers = tuple(_painted_layers(filename, ufo.info.unitsPerEm, picosvg))
+        painted_layers = ()
```

Comment:
perhaps add a comment that `painted_layers` is unused for untouchedsvg output. We could make painted_layers optional and set the value to `None` if we haven't extracted any painted layers. An empty tuple may still be confused for a valid (although not very useful) input. Whereas None would blow up if one attempts to, e.g. iterate over it to get some painted_layers.

Reply:
Good point, changing to None

Reply:
A property that raised an error saying there are no layers for untouched formats, you done goofed would be even nicer but that doesn't fit the tuple well